packages/data-point/README.md

```diff
@@ -1188,6 +1189,50 @@ constant(value:*):*
   ```
 </details>
 
+### <a name="reducer-default">withDefault</a>
+
+The **withDefault** reducer adds a default value to any reducer type. If the reducer resolves to `null`, `undefined`, `NaN`, or `''`,
+the default is returned instead.
```

Comment:
The default value overwrites `null`, `undefined`, `NaN`, `''` (but it does not overwrite `0` or `false`). If anybody has opinions about this I'm open to changing it.

Reply:
perfect!